### ISSUE TITLE
Media library sync with item blaster issue.

### ISSUE THREAD

**Issue by slapikv** [JeffDarchuk/SitecoreSidekick#69]
Hello, recently, we have faced an issue when tried to sync media library items via sidekick content migrator with the enabled "Item blaster" option - the result was that the item was without a preview icon in the media library. Once "Item blaster" was disabled the sync was done successfully, but only for the new items old still were without preview - during a quick check I have found that the preview URL is correct but for some reason, the preview image was not created/generated.
![image](https://user-images.githubusercontent.com/17311014/105471559-e967f780-5cab-11eb-93e9-b823ef002139.png)
We have checked on syncing via Sitecore 9 to 9, Sitecore 10 from 9, the result is the same - no preview image if "Item blaster" option is enabled.

**Comment by JeffDarchuk** [comment]
there isn't a concept of an explicit preview image in Sitecore, the thumbnail is a resized rendering of the original image data.  Are you able to download and view a correct image on the items missing the preview rendering? if not there could have been some image data corruption that happened at some point.  I would try clearing caches, restarting the CM server, rebuilding the link database or running a clean database operation and see if that makes any difference.

**Comment by slapikv** [comment]
In my case, it is not attached:
![image](https://user-images.githubusercontent.com/17311014/105511634-2ac6ca00-5ce1-11eb-9d5e-e394e62b1f15.png)
Sidekick shows that it is synced: 
![image](https://user-images.githubusercontent.com/17311014/105511949-9315ab80-5ce1-11eb-9868-f4277edfaf74.png)
After sync using content migrator, I see it in the skipped tab.

**Comment by JeffDarchuk** [comment]
try running a sync with the ignore rev id toggle

**Comment by slapikv** [comment]
Tried several times - no result if the item does exist in the content tree, deleted image item and synced again - image has appeared in the media field. 
After that tried several times with item blaster enabled the next scenario: delete image item, sync - each time media field was filled in,  the preview has appeared as soon as I cleared the browser cache, forgot that we enabled cache for shell and speak. It seems like the issue is intermittent, lets close this case for now maybe, because a workaround has been found.

Thank you very much for your help!
